app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AndroidNFCAdapter.java

```diff
@@ -0,0 +1,29 @@
+package com.nervousfish.nervousfish.modules.pairing;
+
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+import com.nervousfish.nervousfish.service_locator.IServiceLocatorCreator;
+import com.nervousfish.nervousfish.service_locator.ModuleWrapper;
+
+/**
+ * An adapter to the default Android bluetooth library
+ */
+public final class AndroidNFCAdapter implements INFCHandler {
```

Comment:
Is it useful to have the NFC adapter and the QR adapter in if we are not busy with those implementations yet?